src/components/BooksForm.js

```diff
@@ -34,16 +34,16 @@ const BooksForm = ({addNewBook}) => {
     }
 
     return (
-        <form style={{display: 'flex', justifyContent: 'center'}}>
+        <form onSubmit={handleSubmit} style={{display: 'flex', justifyContent: 'center'}}>
```

Comment:
Please remove the inline styling as pointed out by the previous reviewer. This is a bad CSS practice.

Reply:
I thing the changes are already made,
Thanks for the review

Reply:
I think the changes are already made,
Thanks for the review